Index: scheme48-1.9.2/configure.in
===================================================================
--- scheme48-1.9.2.orig/configure.in	2012-12-27 17:46:24.000000000 +0100
+++ scheme48-1.9.2/configure.in	2018-08-15 08:52:37.853151564 +0200
@@ -22,20 +22,20 @@
      AC_CANONICAL_HOST
      dnl set the cross-compile flag before we try anything.
      AC_TRY_RUN([int main() { return 0;}], [], [], [true])
-     S48_CC
-     S48_PICK_GC
-     S48_ENABLE_FORCE_32BIT
-     S48_DETERMINE_BITS_PER_BYTE
-     S48_DETERMINE_POINTER_SIZE
-     S48_DETERMINE_BIT_SUFFIX
-     S48_ENABLE_UNIVERSAL_BINARY
-     S48_OSX_ARCH_FLAGS($S48_BUILD_UNIVERSAL_P,$S48_FORCE_32_P)
-     S48_PIC_OPTION_NAME
-     S48_CREATE_BUILD_DIRS
-     S48_LDFLAGS
+dnl     S48_CC
+dnl     S48_PICK_GC
+dnl     S48_ENABLE_FORCE_32BIT
+dnl     S48_DETERMINE_BITS_PER_BYTE
+dnl     S48_DETERMINE_POINTER_SIZE
+dnl     S48_DETERMINE_BIT_SUFFIX
+dnl     S48_ENABLE_UNIVERSAL_BINARY
+dnl     S48_OSX_ARCH_FLAGS($S48_BUILD_UNIVERSAL_P,$S48_FORCE_32_P)
+dnl     S48_PIC_OPTION_NAME
+dnl     S48_CREATE_BUILD_DIRS
+dnl     S48_LDFLAGS
      AC_ISC_POSIX
      AC_SYS_LARGEFILE
-     S48_INLINE
+dnl     S48_INLINE
      AC_PROG_INSTALL
      AC_CHECK_LIB(m, main)
      AC_CHECK_LIB(dl, main)
@@ -44,8 +44,8 @@
      AC_CHECK_LIB(gen, main)
      AC_CHECK_LIB(socket, main)
      AC_CHECK_LIB(sun, getpwnam)
-     S48_POSIX_LIBC
-     S48_TM_GMTOFF
+dnl     S48_POSIX_LIBC
+dnl     S48_TM_GMTOFF
      AC_CONST
      AC_MSG_CHECKING(for socklen_t)
      AC_TRY_COMPILE([#include <sys/types.h>
@@ -74,7 +74,7 @@
      AC_CHECK_FUNCS(gettimeofday ftime select setitimer sigaction execvpe)
      AC_CHECK_HEADERS(poll.h)
      AC_CHECK_FUNCS(poll)
-     S48_GLIB
+dnl     S48_GLIB
      AC_CHECK_FUNCS(dlopen)
      AC_CHECK_FUNCS(socket chroot)
      AC_CHECK_FUNC(strerror,
@@ -107,11 +107,11 @@
                            dnl Do not have `__environ'
                                [AC_MSG_WARN([no environ variable found])
                                 AC_LIBOBJ([c/fake/environ.o])])])
-     S48_IEEE_ENDIANNESS($S48_BUILD_UNIVERSAL_P)
-     S48_NATIVE_CODE
-     S48_DYNAMIC_EXTERNALS
-     S48_PTHREADS
-     S48_MISALIGNED_DOUBLES
+dnl     S48_IEEE_ENDIANNESS($S48_BUILD_UNIVERSAL_P)
+dnl     S48_NATIVE_CODE
+dnl     S48_DYNAMIC_EXTERNALS
+dnl     S48_PTHREADS
+dnl     S48_MISALIGNED_DOUBLES
      AC_SUBST(CFLAGS)
      AC_SUBST(CPPFLAGS)
      AC_SUBST(LDFLAGS)
